helm/forgerock-metrics/README.md

```diff
@@ -57,10 +68,10 @@ A PrometheusRules CRD has been included in the Helm chart which includes the fr-
 * Deployed ForgeRock application in Google Cloud cluster.
 * Kubectl authenticated to cluster.
 * Separate namespace prepared for deploying Prometheus/Grafana
-(Use namespace that is different to where your application is running).
+(Use namespace that is different to where your application is running, default is 'monitoring').
 
 ### Prepare for deployment
-* cd to monitoring folder within forgeops repo.
+* cd to bin folder within forgeops repo.
```

Comment:
s/to bin folder within forgeops repo/to the bin folder in your forgeops repo clone/